tests/ui/imports/reexports.rs

```diff
@@ -5,9 +5,12 @@ mod a {
     mod foo {}
 
     mod a {
-        pub use super::foo; //~ ERROR cannot be re-exported
+        pub use super::foo;
+        //~^ ERROR cannot be re-exported
```

Comment:
If it's possible to easily detect erroneous imports using something like `get_import_res() == Res::Err`, then we should skip the unused import warnings for such imports.

Reply:
Why do you assume this import points to ﻿`Res::Err`? This single import has a determined binding, and it points to `﻿TypeNS(root::a::foo), ValueNS(root::a::foo), MacroNS(None)`.

Perhaps you're proposing that in the following case, `use a::b` should not trigger an ﻿`unused_imports` warning?

```rs
use non_existent_mod as a;
use a::b; 
```

Reply:
Okay, if the resolution for "cannot be re-exported" items is not `Res::Err` then it's fine to keep this as is.